store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9929,4 +9929,10 @@ TODO: delete them permanently from here
   <defaultCOSValue>30</defaultCOSValue>
   <desc>Default calendar resolution preference</desc>
 </attr>
+<attr id="5000" name="zimbraGlobalExternalStoreConfig" type="string" cardinality="multi" optionalIn="globalConfig" since="9.1.0">
```

Comment:
why we are using multi-value attributes?  we will store info in json object and it would be a single json object - right?